eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -4,227 +4,124 @@ languages:
-    Please note `Basic` tier is unsupported. After creating the Azure Event Hub, you
-    can create your own Consumer Group or use the default "$Default" Consumer Group.
-
-1.  Create [Azure Storage][create-azure-storage] for checkpoint use.
-
-1.  Add Role Assignment for Event Hub and Storage Account. See
-    [Managed identities for Azure resources with Event Hubs][role-assignment]
-    to add role assignment for Event Hub, Storage Account is similar.
-
-    - Event Hub: assign `Contributor` role for managed identity.
-    - Storage Account: assign `Storage Account Key Operator Service Role` 
-      role for managed identity.
-
-##### Update MSI related properties
-
-1.  Update [application-mi.yaml][application-mi.yaml]
-    ```yaml
-    spring:
-      cloud:
-        azure:
-          msi-enabled: true
-          client-id: [the-id-of-managed-identity]
-          resource-group: [resource-group]
-          # Fill subscription ID copied from portal
-          subscription-id: [subscription-id]
-          eventhub:
-            namespace: [eventhub-namespace]
-            checkpoint-storage-account: [checkpoint-storage-account]
-            checkpoint-container: [checkpoint-container]
-        stream:
-          function:
-            definition: consume;supply
-          bindings:
-            consume-in-0:
-              destination: [eventhub-name]
-              group: [consumer-group]
-            supply-out-0:
-              destination: [the-same-eventhub-name-as-above]
-    ```
-    > We should specify `spring.profiles.active=mi` to run the Spring Boot application. 
-      For App Service, please add a configuration entry for this.
-
-##### Redeploy Application
-
-If you update the `spring.cloud.azure.managed-identity.client-id`
-property after deploying the app, or update the role assignment for
-services, please try to redeploy the app again.
-
-> You can follow 
-> [Deploy a Spring Boot JAR file to Azure App Service][deploy-spring-boot-application-to-app-service] 
-> to deploy this application to App Service
-
-#### Enable auto create
-
-If you want to auto create the Azure Event Hub and Azure Storage account instances,
-make sure you add such properties 
-(only support the service principal and managed identity cases):
-
-```yaml
-spring:
-  cloud:
-    azure:
-      subscription-id: [subscription-id]
-      auto-create-resources: true
-      environment: Azure
-      region: [region]
-```
-
-#### Enable sync message
-To enable message sending in a synchronized way with Spring Cloud Stream 3.x, 
-azure-spring-cloud-stream-binder-eventhubs supports the sync producer mode to get responses for sent messages. 
-By enabling following configuration, you could use [StreamBridge][StreamBridge] for the synchronized message producing.
-
-```yaml
-spring:
-  cloud:
-    stream:
-      eventhub:
-        bindings:
-          supply-out-0:
-            producer:
-              sync: true
-```
-
 #### Using Batch Consuming
```

Comment:
This section should not be listed between different credential configuration, you could put it in a new section like "Enhancement" after section of "Example". And don't delete origin features like "#### Enable sync message"

Reply:
updated